portal-web/test/functional/com/liferay/portalweb/macros/AppManager.macro

```diff
@@ -14,6 +14,64 @@ definition {
 		AssertConfirm(value1 = "Are you sure you want to deactivate this?");
 	}
 
+	macro filtersApp {
+		var key_filterName = "${filterName}";
+
+		SelectFrame.selectFrame(locator1 = "IFrame#MARKETPLACE_STORE_PORTLET_IFRAME");
```

Comment:
Try if adding the step to initialize Liferay Event before the select iFrame can avoid the failure `VERIFICATION_WARNING: Event "modalIframeLoaded" with attribute "src" and value "dialog-iframe-popup" not fired.
` in report. 
`WaitForLiferayEvent.initializeLiferayEventLog();`

Reply:
run report and see

Reply:
Use SelectFrame.selectFrameNoLoading